topics/imaging/tutorials/process-image-bioimageio/tutorial.md

```diff
@@ -0,0 +1,182 @@
+
+## Model-specific example
+
+This example illustrates the type of information that is both useful for understanding the model’s biological context and necessary for using the Galaxy tool — specifically, the input axes and input size parameters.
+
+<details>
+<summary><strong>🧬 NucleiSegmentationBoundaryModel</strong></summary>
+
+This model segments nuclei in fluorescence microscopy images. It predicts boundary maps and foreground probabilities for nucleus segmentation, primarily in images stained with DAPI. The outputs are designed to be post-processed with methods such as Multicut or Watershed to achieve instance-level segmentation.
+
+- **Imaging modality**: Fluorescence microscopy  
+- **Task**: Nucleus segmentation (boundary-aware)  
+- **Input axes**: `bcyx`  
+- **Input size**: `256,256,1,1`  
+- **Model link**: [View on BioImage.IO](https://bioimage.io/#/r/ilastik/stardist_dsb_training_data)  
+- **Citation**: [10.5281/zenodo.5764893](https://doi.org/10.5281/zenodo.5764893)
+
+</details>
+
+> <tip-title> Where to find this information on BioImage.IO </tip-title>
+>
+> You can find similar details for other models directly on [BioImage.IO](https://bioimage.io) by viewing each model’s card. Look under the “inputs” section of the RDF file to find the required `axes` and `input size` values. These parameters are essential for running the model correctly in Galaxy.
+>
+{: .tip}
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+>
+> 2. Download the following image and import it into your Galaxy history.  
+>    For the purpose of this tutorial, we will use one image to test only one of the 11 available models:
+>
+>    - [`test_image_nuclei.tif`](../../images/process-image-bioimageio/input_nucleisegboundarymodel.png)
+>
+>    If you are importing the image via URL:
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    If you are importing the image from the shared data library:
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets appropriately if needed (e.g. `"BioImage.IO model"`, `"Test image"`)
+>
+> 4. Confirm the datatypes are correct (`pt` for the model, `tiff` or `png` for the image)
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Import the BioImage.IO model from the Galaxy file repository:
+>
+>    - Click on **Upload Data**
+>    - Go to the **Choose remote files** tab
+>    - Navigate to: `ML models` → `bioimaging-models`
+>    - Select the desired model file (for this tutorial, choose `nucleisegmentationboundarymodel.pt`)
+>    - Click **Import** to add it to your history
+>
+>    If you are importing the model from the shared data library:
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
```

Comment:
There is something wrong with the formatting here:
<img width="838" alt="Bildschirmfoto 2025-04-02 um 14 28 05" src="https://github.com/user-attachments/assets/8b4f1b09-5d62-4a42-a600-8b683d4d9055" />